src/storage/src/hummock/test_utils.rs

```diff
@@ -219,9 +219,16 @@ pub async fn gen_test_sstable(
     gen_test_sstable_inner(opts, sst_id, kv_iter, sstable_store, CachePolicy::NotFill).await
 }
 
-/// The key (with epoch 0) of an index in the test table
+pub fn prefixed_key<T: AsRef<[u8]>>(key: T) -> Bytes {
+    let mut buf = Vec::new();
+    buf.put_u32(0);
```

Comment:
add some comments for  `prefix` 0 ?

Reply:
I just chose a random number. Anything to watch out about this dummy table id?

Reply:
`TableId::default()` , the default value of table_id is 0